app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -367,7 +379,13 @@ def additional_response_formats(format)
     end
 
     def save_permissions
-      @saved_permissions = curation_concern.permissions.map(&:to_hash)
+      @saved_permissions =
+        case curation_concern
+        when ActiveFedora::Base
+          curation_concern.permissions.map(&:to_hash)
+        else
+          Hyrax::AccessControl.for(resource: curation_concern).permissions
+        end
     end
 
     def permissions_changed?
```

Comment:
Does this method need to be updated to work with Hyrax::Resource objects as well?

Reply:
Nevermind.  It looks like that will come later when dealing with works with file sets.

Reply:
:+1: